### PR TITLE
Connect hosts to api

### PR DESCRIPTION
**Pull request by jtomasek**
Populates HostsTable with actual data from API.

Work in progress
- needs to add reselect for cached selectors
- needs to add loading status tracking
- clean up HostsExample from App.tsx

Depends on #22

### CODE REVIEW

File: src/models/hosts.ts [hunk 1]
```diff
@@ -1,3 +1,9 @@
 export interface Host {
-  id: string;
+  name: string;
+  ip: string;
```
Comment (honza) [inline src/models/hosts.ts]: The types of the fields of `Host` should match whatever the server returns.

File: src/components/BaremetalInventory.tsx [hunk 2]
```diff
@@ -1,37 +1,66 @@
...
-      variant={PageSectionVariants.light}
-      isMain
-      style={{ padding: 0 }}
-    >
-      <HostsTable />
-    </PageSection>
-  </Fragment>
-);
+interface Props {
+  hostRows: string[][];
```
Comment (honza) [inline src/components/BaremetalInventory.tsx]: Let's use a more concrete type here instead of `string[][]`

File: src/components/HostsTable.tsx [hunk 3]
```diff
@@ -1,8 +1,22 @@
 import React, { FC, Fragment } from 'react';
 import { Table, TableHeader, TableBody } from '@patternfly/react-table';
+import { Bullseye } from '@patternfly/react-core';
 
-const HostsTable: FC = (): JSX.Element => {
-  const headerStyle = { position: 'sticky', top: 0, background: 'white' };
+interface Props {
+  hostRows: string[][];
```
Comment (honza) [inline src/components/HostsTable.tsx]: Same as above

**Comment by jtomasek** [comment]
Rebased on top of `master`

File: src/components/HostsTable.tsx (hunk 3 above)
Comment (jtomasek) [reply src/components/HostsTable.tsx]: done

File: src/components/BaremetalInventory.tsx (hunk 2 above)
Comment (jtomasek) [reply src/components/BaremetalInventory.tsx]: done

File: src/models/hosts.ts (hunk 1 above)
Comment (jtomasek) [reply src/models/hosts.ts]: done

**Pull request merged**